umh-core/pkg/communicator/topicbrowser/communicator_metadata_strip_test.go

```
@@ -0,0 +1,193 @@
+		data, err := comm.GetSubscriberData(false)
+		Expect(err).NotTo(HaveOccurred())
+
+		cacheBytes, ok := data.UnsBundles[0]
+		Expect(ok).To(BeTrue(), "bootstrap bundle (index 0) must be present")
+
+		var decoded tbproto.UnsBundle
+		Expect(proto.Unmarshal(cacheBytes, &decoded)).To(Succeed())
+
+		wiredEntries := decoded.GetUnsMap().GetEntries()
+		Expect(wiredEntries).ToNot(BeEmpty(), "bootstrap bundle must carry its topic map")
+		for hash, entry := range wiredEntries {
```

golangci-lint (2.12.2)

[error] missing whitespace above this line (invalid statement above range)

(wsl_v5)